src/ByteSizeLib/BinaryByteSize.cs

```diff
@@ -5,53 +5,68 @@ namespace ByteSizeLib
 {
     public partial struct ByteSize
     {
+        /// <summary>Number of bytes in 1 kibibyte.</summary>
```

Comment:
I found that redundant since the current IntelliSense shows the value of the constant without any additional work. 
![image](https://user-images.githubusercontent.com/196368/140623112-63b91674-9e03-4c1c-83cb-e4ebd7dd0bf6.png)